### DISCUSSION TITLE
建议为 谷歌翻译 新增 自定义网址的功能, 就像 openAI的设置一样, 以便免于搭梯子

### DISCUSSION THREAD

**Discussion by AwesomeYuer** [immersive-translate/immersive-translate#386] [Ideas]
### 新功能 | New Feature

1. 建议为 谷歌翻译 新增 自定义网址的功能, 就像 openAI的设置一样

### 为什么您想要添加这些功能 | Why you want to add these features

1. 以便免于搭梯子
2.
3.

**Comment by wenxywang** [comment]
连不上互联网你是怎么上github的

**Comment by versun** [comment]
这个当前在开发者设置里，可以手动修改 user config 来实现 (插件设置->开发者设置->Edit Full User Config)
目前还没有做界面，因为可能会对普通用户造成困扰。
找到字段translationServices，添加即可(注意，多个Services直接要有逗号分隔)
```
"translationServices": {
 "deepl": {
      "authKey": "111010100101"
    },
  "google": {
    "apiUrl":"https://your-domain.com"
  }
}
```
文档： https://immersive-translate.owenyoung.com/advanced#%E7%BF%BB%E8%AF%91%E6%9C%8D%E5%8A%A1%E9%85%8D%E7%BD%AE

**Reply by AwesomeYuer** [reply]
谢谢啊
